### DISCUSSION TITLE
Omni key opening screen cap?

### DISCUSSION THREAD

**Discussion by Nchi** [Lailloken/Exile-UI#173] [Q&A]
I am just starting to try to use the omnikey for cheatsheets- I thought it was automated for some cheat sheets too but nothing is really popping up, alt ctrl c on betrayal opens screen cap mode with a prompt from llkui to name or abort screen cap- same with right click + middle click on sanctum screen? Not sure whats going on with the feature so wasn't sure if I should open an issue or its just misuse on my part, thanks

**Comment by Lailloken** [comment]
The screen-capping is there to "teach" the script how to read the screen correctly. You screen-cap parts of the screen (syndicate-member names in Betrayal, room names for sanctum) and then tell the script what the text says. The next time you click the omni-key on the betrayal board or sanctum map, the script will know what is on screen and show you the appropriate information.

This approach requires a little bit of work from the user, but this makes the screen-reading/image-recognition really fast and accurate. It would be really difficult for me to implement screen-reading that works this well for every screen resolution, graphics filters, gamma settings, etc.

**Comment by Nchi** [comment]
I should have mentioned that I have done that for sanctum and betrayal, so the expectation is to fix all of the red marked caps first?

So I restarted the script- betrayal now works as expected with a simple middle click, same with beast alter, but sanctum isnt reacting to the simple middle click even with the test being positive- here is where right click + middle mouse is opening the screen cap still

**Reply by Lailloken** [reply]
Ah, I see. Sanctum is a bit different: you hold the omni-key and hover the rooms. I didn't want to require closing the tooltips with ESC because they only have so little text.

**Reply by Nchi** [reply]
Still not getting any type of response from holding middle click :( maybe something did break then- whats with the screen cap opening on rclick+mclick too?

**Comment by Nchi** [comment]
Ah I have found the disconnect, some labeling on the box that pops up would go a long way

**Reply by Lailloken** [reply]
Sorry about that, I forgot to mention in the release notes that the video-clip shows which parts need to be screen-capped.
